examples/hooks/RecordHooks/conditionallyEmptyField.spec.ts

```diff
@@ -0,0 +1,51 @@
+import { NumberField, TextField, Sheet, Workbook } from '@flatfile/configure'
+import { SheetTester } from '../../../src/utils/testing/SheetTester'
+
+//this is a basic example of how to modify a field based on other fields
+const testSheet = new Sheet(
+  'Test Sheet',
+  {
+    paymentStatus: TextField(),
+    amount: NumberField(),
```

Comment:
Side note. we are encouraging all Fields to be called with at least an empty object `{}` so this becomes
`NumberField({})`